core/src/main/java/org/apache/calcite/adapter/enumerable/RexImpTable.java

```diff
@@ -419,6 +440,59 @@ public Expression implement(RexToLixTranslator translator,
         new SequenceImplementor(BuiltInMethod.SEQUENCE_NEXT_VALUE.method),
         false);
 
+    // Json Operators
+    defineMethod(JSON_VALUE_EXPRESSION,
+        BuiltInMethod.JSON_VALUE_EXPRESSION.method, NullPolicy.STRICT);
+    defineMethod(JSON_STRUCTURED_VALUE_EXPRESSION,
+        BuiltInMethod.JSON_STRUCTURED_VALUE_EXPRESSION.method, NullPolicy.STRICT);
+    defineMethod(JSON_API_COMMON_SYNTAX, BuiltInMethod.JSON_API_COMMON_SYNTAX.method,
+        NullPolicy.NONE);
+    defineMethod(JSON_EXISTS, BuiltInMethod.JSON_EXISTS.method, NullPolicy.NONE);
+    defineMethod(JSON_VALUE_ANY, BuiltInMethod.JSON_VALUE_ANY.method, NullPolicy.NONE);
+    defineMethod(JSON_QUERY, BuiltInMethod.JSON_QUERY.method, NullPolicy.NONE);
+    defineMethod(JSON_OBJECT, BuiltInMethod.JSON_OBJECT.method, NullPolicy.NONE);
+    aggMap.put(JSON_OBJECTAGG_NULL_ON_NULL, new Supplier<AggImplementor>() {
```

Comment:
Could you please use `() -> ` syntax here?
Or something like `aggMap.put(JSON_OBJECTAGG_NULL_ON_NULL, JsonObjectAggImplementor.for(BuiltInMethod.JSON_OBJECTAGG_ADD_ABSENT_ON_NULL))`

Reply:
+1 thanks.